ts/components/ui/FooterButtons.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, NativeBase, Text, View } from "native-base";
+import { connectStyle } from "native-base-shoutem-theme";
+import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
+import * as React from "react";
+
+type OwnProps = {
+  title: string;
+};
+
+type Props = Readonly<{
+  leftButton: NativeBase.Button & OwnProps;
+  rightButton: NativeBase.Button & OwnProps;
+  inlineHalf: boolean;
+  inlineOneThird: boolean;
+}>;
+
+/**
+ * Implements a component that show 2 buttons in footer with select style (inlineHalf | inlineOneThird)
+ */
+class FooterButtons extends React.Component<Props> {
```

Comment:
I think FooterButtons is not a good name because you are not only rendering the Buttons but also the entire footer (`<View footer={true}>`), so i think FooterWithButtons is better.

Reply:
I agree